.circleci/config.yml

```diff
@@ -97,6 +97,77 @@ commands:
           event: pass
           template: basic_success_1
 
+  cf_deploy_review:
+    steps:
+      - run:
+          name: "Setup CF CLI"
+          command: |
+            curl -L -o cf.deb --retry 3 'https://packages.cloudfoundry.org/stable?release=debian64&version=7.2.0&source=github-rel'
+            file cf.deb
+            sudo dpkg -i cf.deb
+            cf -v
+            cf api "$CF_ENDPOINT"
+            cf auth "$CF_USER" "$CF_PASSWORD"
+            cf add-plugin-repo CF-Community https://plugins.cloudfoundry.org
+            cf install-plugin app-autoscaler-plugin -r CF-Community -f
+            cf target -o "$CF_ORG" -s development
+      - run:
+          name: "Fetch existing manifest"
+          command: |
+            cf create-app-manifest "$CF_APP-dev" -p deploy_manifest.yml
+      - run:
+          name: "Update app manifest"
+          command: |
+            sed -i  "s/instances: [0-9]\+/instances\: 1/" deploy_manifest.yml
+      - run:
+          name: "Push review app"
+          command: |
+            export DOCKER_IMAGE=tariff-frontend
+            export DOCKER_TAG=dev-${CIRCLE_SHA1}
+            export PR_NUMBER=$(echo $CIRCLE_PULL_REQUEST | sed 's|https://github.com/||' | cut -d '/' -f 4)
+            export REVIEW_URL_ID="pr${PR_NUMBER}"
```

Comment:
Do we want to include the branch for semantic helpfuls?

Reply:
My take would be a definite no - ultimately we've already got a unique identifier (the PR number) - adding something else in just makes the URL much more complicated so that leads to the question what do we gain by it?